dataactbroker/README.md

```diff
@@ -555,14 +555,12 @@ Example output:
 
 ## Test Cases
 
-To run the broker API unit tests, navigate to the main project folder (`data-act-broker`) and type the following:
+To run the broker API unit tests, navigate to the main project folder (`data-act-broker-backend`) and type the following:
```

Comment:
Good catch! This made me realize something else...the runTests.py command will not run both the broker and validator tests, which is probably what we want. However, the validator tests won't run unless you're actually in the tests folder (we should probably fix that in a tech debt task at some point). I'll push a change that updates these directions accordingly.